.github/actions/store_artifacts/action.yml

```diff
@@ -31,7 +31,7 @@ runs:
       id: copy_artifacts
       shell: ${{ runner.os == 'Windows' && 'pwsh' || 'bash' }}
       run: >-
-        python ${{ env.ACTION_PATH }}/store_artifacts.py
+        python3 ${{ env.ACTION_PATH }}/store_artifacts.py
```

Comment:
The same as in https://github.com/openvinotoolkit/openvino/pull/25234#discussion_r1694962496.